packages/x-license/src/verifyLicense/verifyLicense.ts

```diff
@@ -169,6 +178,14 @@ export function verifyLicense({
     return { status: LICENSE_STATUS.Invalid };
   }
 
+  if (license.planVersion === 'initial') {
```

Comment:
Would a license without `planVersion` fall into this condition?
It would be great to have a test for this case.

Reply:
Yes, since we set 'initial' by default when decoding the license.

Reply:
Interesting. I guess we'd need to request the users to use the latest x-license package. 

Is there any known incompatibility with the latest package and older versions of the components package?

Reply:
> we'd need to request the users to use the latest x-license package.

Why would that be?
Could you describe the scenario that would not work (which version of `@mui/x-license`, which version of`@mui/x-data-grid-pro` or equivalent and which format of license)?

Reply:
This simply cannot happen ... changes to the logic are all restricted to the `@mui/x-license` package. Any of the commercial packages imports `useLicenseVerifier` from there.

Even using a new license (including the `planVersion`) works with an old version of the license package, since when decoding the license we simply won't add this info to the license object

Reply:
But what happens if a user with the new license tries to use an old version of the component? Can older x-license packages decode the new key?
That's the case I'm thinking they might need to use the latest x-license package.

Reply:
Comment above^^

What happens is basically that this piece of info gets dropped in the decoding process

Reply:
>  Can older x-license packages decode the new key?

Yes, it would just ignore the new property `PV` and verify it as it would verify a license generated today (or one year ago)
It was the goal of the new property, to make sure it's backward compatible in this scenario :+1: 
The new license key is basically and old license key plus the `PV`, we do not touch any existing property.